pose_estimation_node:
  ros__parameters:
    max_pitch: 30
    min_match_altitude: 50
    attitude_deviation_threshold: 10
    pose_estimator_endpoint: 'http://localhost:8090/predictions/loftr'
